app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,8 @@
+class StaticPagesController < ApplicationController
+  def home
+    @food_product = Product.order_created_at_desc.category_by(Settings.category.food).page(params[:food_page]).per Settings.paginate.maximum_page
+    @drink_product = Product.order_created_at_desc.category_by(Settings.category.drink).page(params[:drink_page]).per Settings.paginate.maximum_page
+    @food_count = Product.category_by(Settings.category.food).count
```

Comment:
- Dùng size thay  count.
- Phân biệt size, count, length

Reply:
count: Dùng câu lệnh sql Select count() để truy vấn ra số lượng của activerecord và count không được lưu trữ trong đối tượng được gọi
length: đếm số lượng mà không sử dụng thêm câu truy vấn nếu đối tượng đã được nạp, nó nạp toàn bộ object vào bộ nhớ để thực hiện đếm
size: khi object được load, nó sẽ thực hiện chính object đó trong bộ nhớ,nếu object chưa đc load sẽ thực hiện câu lệnh sql